pageserver/src/waldecoder.rs

```diff
@@ -1113,7 +1113,7 @@ pub fn decode_wal_record(checkpoint: &mut CheckPoint, record: Bytes) -> DecodedW
                 blocks.push(blk);
             }
         } else {
-            assert!(false);
+            panic!()
```

Comment:
Whoever wrote `assert!(false)`, feel free to chime in here with the message that should be in the panic.